notebooks/Tables.ipynb

```diff
@@ -30,6 +30,7 @@
     "from ibm_watson import ApiException\n",
```

Comment:
<p>Averaging together different items with different units doesn't really make sense. I'd recommend doing max of the two tax rates to show aggregation along an axis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>